### PR TITLE
Don't warn about students working when deleting unopened assignments

### PR DESCRIPTION
**Pull request by nathanstitt**
Work was done via the the "get the code to a place where you can make the easy change, then make it" philosophy.

Also, I'm surprised we don't have a "isPlanOpened" helper somewhere.  Maybe I just missed it though?

screen shot 2016-09-06 at 4 44 14 pm

### CODE REVIEW

File: tutor/src/helpers/plan.coffee [hunk 1]
```diff
@@ -27,4 +27,9 @@ PlanHelper =
     {isPublishing, publishStatus}
 
 
+  isPlanOpen: (plan) ->
+    now = moment(TimeStore.getNow())
+    !! _.find plan?.tasking_plans, (tasking) -> moment(tasking.opens_at).isBefore(now)
```
Comment (pandafulmanda) [inline tutor/src/helpers/plan.coffee]: `isPlanOpen`/something similar does exist.  It's just my fault that it's in a weird place [in the `PlanMixin`](https://github.com/openstax/tutor-js/blob/master/tutor/src/components/task-plan/plan-mixin.cjsx#L25-L27).  This does get pulled in to the parents of the `PlanFooter`, so the `state.isVisibleToStudents`  could be passed in to `PlanFooter` and `DeleteLink` as a `prop`.

**Comment by pandafulmanda** [comment]
yayyy for breaking the footer into smaller components!

File: tutor/src/helpers/plan.coffee (hunk 1 above)
Comment (nathanstitt) [inline tutor/src/helpers/plan.coffee]: Yeah, I'd prefer to have all the components in the tree use the same logic.  I'll pass `isVisibleToStudents` into Footer and and then onto the child components.

Comment (pandafulmanda) [inline tutor/src/helpers/plan.coffee]: if this is no longer used, could we remove?

Comment (nathanstitt) [inline tutor/src/helpers/plan.coffee]: heh, yeah. That was an oversight - I meant to remove when I switched to the `isVisibleToStudents` prop.  Thanks!

File: tutor/src/components/task-plan/reading.cjsx [hunk 2]
```diff
@@ -132,6 +132,7 @@ ReadingPlan = React.createClass
 
     footer = <PlanFooter
       id={id}
+      isVisibleToStudents={@state.isVisibleToStudents}
```
Comment (pandafulmanda) [inline tutor/src/components/task-plan/reading.cjsx]: Should these get passed into external and/or event footers as well?

**Pull request merged**